ocaml/quicktest/jbuild

```diff
@@ -31,8 +31,14 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
   (public_name quicktestbin)
   (package xapi)
   (flags (:standard -bin-annot %s -warn-error +a-3-4-6-9-27-28-29-52))
-  (libraries (
-   xapi_internal
-  ))
+  (libraries
+    (alcotest
+     stdext
```

Comment:
Please remove `stdext`

Reply:
But the `Stdext` module is actually used by the code, so will that work?

Reply:
Please get rid of it from the code. It should be a fairly mechanical replacemenf of `open Stdext open ...` with `open Xapi_stdext_...`